schema/snapcraft.yaml

```diff
@@ -57,16 +57,6 @@ properties:
     uniqueItems: true
     items:
       - type: string
-  license:
-    type: string
-    description: path to a license file.
-  license-agreement:
-    type: string
-    description: action to take on licensing.
-    enum:
-      - explicit
-  license-version:
-    description: license version (used to re-trigger an agreement action).
```

Comment:
Won't removing this from the schema cause the `snapcraft.yaml`s out there that are using this to actually break? What if we left them along with the deprecation warning, but rephrased the deprecation warning to say "this doesn't actually do anything anymore"?